recipes/default.rb

```diff
@@ -104,6 +104,8 @@
   command "#{node['squid']['package']} -Nz"
   action :run
   creates ::File.join(node['squid']['cache_dir'], '00')
+  notifies :stop, "service[#{squid_service_name}]", :before
```

Comment:
Before is pretty fragile and entirely broken in things like why-run. If there's a way to do it without this that'd be great

Reply:
The other idea would be to drop initialization entirely since on at least debian the init script will create it if it doesn't exist. But I don't know if that's true across the board.

Reply:
On the upside, `:before` is doing the trick and all tests are passing and there is [some precedence](https://github.com/sous-chefs/mariadb/blob/9f03755f168c59294507e31cb27f551e8eb1967d/resources/server_install.rb#L101-L103) for using it within this codebase.